azure-pipelines.yml

```diff
@@ -0,0 +1,173 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+
+pr: none
+
+variables:
+  imageName: 'apply-for-postgraduate-teacher-training'
+  debug: true
+
+stages:
+- stage: build_test_release
+  displayName: 'Build, Test & Release'
+  jobs:
+  - job: build_and_test_docker_image
+    displayName: 'Build & Test Docker Image'
+    pool:
+      vmImage: 'Ubuntu-16.04'
+
+    variables:
+      # Support multiple docker-compose files to allow travis and az pipelines to co exist: https://docs.docker.com/compose/extends/#multiple-compose-files
+      dockerOverride: 'docker-compose -f docker-compose.yml'
+      system.debug: $(debug)
+
+    steps:
+    - script: |
+        GIT_SHORT_SHA=$(echo $(Build.SourceVersion) | cut -c 1-7)
+        docker_path=$(dockerHubUsername)/$(imageName)
+        echo "##vso[build.updatebuildnumber]$GIT_SHORT_SHA"
+        echo "##vso[task.setvariable variable=docker_path;]$docker_path"
+      displayName: 'Set version number'
+
+    - script: |
+        $DOCKER_OVERRIDE build
+        $DOCKER_OVERRIDE up --no-build -d
```

Comment:
Why do we pass `--no-build` when we have just built the image? `docker-compose` will not build unless we pass `--build`?